job_templates/ci_job.xml.em

```diff
@@ -132,12 +131,18 @@ fi
 if [ "$CI_USE_WHITESPACE_IN_PATHS" = "true" ]; then
   export CI_ARGS="$CI_ARGS --white-space-in sourcespace buildspace installspace workspace"
 fi
-export CI_ARGS="$CI_ARGS --ignore-rmw"
-if [ "$CI_USE_CONNEXT_STATIC" = "false" ]; then
+export CI_ARGS="$CI_ARGS --ignore-rmw rmw_connext_dynamic_cpp"
+# TODO(asorbini) `rmw_connext_cpp` is still the default for dashing" and foxy.
+if [ -n "$CI_ROS_DISTRO" -a \( "$CI_ROS_DISTRO" = dashing -o "$CI_ROS_DISTRO" = foxy \) ]; then
+  if [ "$CI_USE_CONNEXTDDS" = "false" ]; then
+    export CI_ARGS="$CI_ARGS rmw_connext_cpp"
+  fi
+else
+  # Always ignore `rmw_connext_cpp` in favor of `rmw_connextdds` for older releases.
```

Comment:
Shouldn't this comment say:

```
# Always ignore `rmw_connext_cpp` in favor of `rmw_connextdds` for newer releases.
```

?  I'm just making sure I understand what is going on here.

(there are several other instances of this comment in the code below)

Reply:
Fixed in #552

Reply:
(Your analysis was correct, my bad for that typo)